daemon/rpc/server.go

```diff
@@ -1,29 +1,26 @@
 package rpc
 
 import (
-	"context"
 	"fmt"
 	"net"
 
-	log "github.com/sirupsen/logrus"
 	"google.golang.org/grpc"
 )
 
-type Server struct {
-	UnimplementedSwapServiceServer
-	Port int
+type Repository interface {
```

Comment:
What's a Repository in this context?

Reply:
Repository will contain a collection of repositories.

```golang
type Repository interface {
   database.SwapInRepository
   database.SwapOutRepository
}
```

So that @markettes and I we can work in different files